src/pages/docs/pages/server-rendering.md

```diff
@@ -108,7 +108,7 @@ A couple of notes:
 
 ### Body
 
-To return just the body of the page, you can use the `getBody` API. You will get access the [compilation](/docs/reference/appendix/#compilation), page specific data, and the incoming request.
+To return just the body of the page, you can use the `getBody` API. You will get access to the [compilation](/docs/reference/appendix/#compilation), page specific data, and the incoming request.
```

Comment:
ah, thanks, but could we keep this is in it's own PR please?  I try to avoid doing multiple unrelated things in the same PR.